.github/workflows/crowdin-ui-download.yml

```diff
@@ -0,0 +1,355 @@
+name: i18n - Download Client UI
+on:
+  workflow_dispatch:
+  schedule:
+    # runs Monday and Wednesday at 12:15 PM UTC
+    - cron: '15 12 * * 1,3'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+  CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
+  CROWDIN_API_URL: 'https://freecodecamp.crowdin.com/api/v2/'
+  CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_CLIENT }}
+
+jobs:
+  i18n-download-client-ui-translations:
+    name: Client
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          token: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+
+      - name: Generate Crowdin Config
+        uses: freecodecamp/crowdin-action@main
+        env:
+          PLUGIN: 'generate-config'
+          PROJECT_NAME: 'client'
```

Comment:
This needs to be renamed. The `client` project refers to our `/learn` client.

Reply:
I need some help providing these with proper Crowdin names. Maybe something like "contribute-ui" for this set of files. And "contribute-content" for the markdown files themselves.